CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to
 
 ### Added
 
+- Add unique constraint to owner address field to allow only one main address
+  per user
+- Add fullname field to address model
 - Add a web hook endpoint to synchronize course runs from a LMS
 - Add a "languages" field to the course run model
 - Add stub dependencies required by mypy
```

Comment:
### Changed

- Rename the "name" field to "title" (avoid confusion with new "fullname" field)
- Rename "main" field to "is_main" as our naming convention for boolean fields